### PR TITLE
Updated contributing documentation

### PR DESCRIPTION
**Pull request by patricklafrance**
- Updated the PNPM workspaces documentation
- A bunch of minor touch ups

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -227,21 +219,21 @@ Before you add a new package, please read the [GSoft GitHub guidelines](https://
...
 pnpm init
 ```
 
 Answer the CLI questions.
 
-Once the *package.json* is generated, please read again the [GSoft GitHub guidelines](https://github.com/gsoft-inc/github-guidelines#npm-package-name) and make sure the package name, author and license are valid.
+Once the `package.json` file is generated, please read again the [GSoft GitHub guidelines](https://github.com/gsoft-inc/github-guidelines#npm-package-name) and make sure the package name, author and license are valid.
 
-Don't forget to add the [npm scope](https://docs.npmjs.com/about-scopes) *"@workleap"* before the package name. For example, if the project name is "foo", your package name should be "@workleap/foo".
+Don't forget to add the [npm scope](https://docs.npmjs.com/about-scopes) `"@workleap"` before the package name. For example, if the project name is "foo", your package name should be `@workleap/foo.
```
Comment (ofrogon) [inline CONTRIBUTING.md:234]:
```suggestion
Don't forget to add the [npm scope](https://docs.npmjs.com/about-scopes) `"@workleap"` before the package name. For example, if the project name is "foo", your package name should be `@workleap/foo`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,44 +13,36 @@ The following documentation is only for the maintainers of this repository.
 
 ## Monorepo setup
 
-This repository is managed as a monorepo that is composed of many npm packages.
+This repository is managed as a monorepo with [PNPM workspace](https://pnpm.io/workspaces) to handle the installation of the npm dependencies and manage the packages interdependencies.
 
-For more information on monorepo:
+It's important to note that PNPM workspace doesn't hoist the npm dependencies at the root of the workspace as most package manager does. Instead, it use an advanced [symlinked node_modules structure](https://pnpm.io/symlinked-node-modules-structure). This means that you'll find a `node_modules` directory inside the packages folders as well as at the root of the repository.
```
Comment (ofrogon) [inline CONTRIBUTING.md:18]:
```suggestion
It's important to note that PNPM workspace doesn't hoist the npm dependencies at the root of the workspace as most package manager does. Instead, it uses an advanced [symlinked node_modules structure](https://pnpm.io/symlinked-node-modules-structure). This means that you'll find a `node_modules` directory inside the packages folders as well as at the root of the repository.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,44 +13,36 @@ The following documentation is only for the maintainers of this repository.
...
-
-It's important to note that PNPM workspace will **link** the npm dependencies at the root of the workspace. This means that there might not be a *node_modules* directory nested in the packages directories. The npm dependencies are installed in a *node_modules* directory at the root of the workspace and a single *pnpm-lock.yaml* file is generated at the root of the workspace.
-
-## Project Overview
+This project is split into two major sections, [packages/][packages/] and [templates/](templates/).
 
 ### Packages
 
-This project is split into two major sections, `packages/` and `templates/`.
+Under [packages/][packages/] we have two tools that are deployed on [NPM](https://www.npmjs.com/).
```
Comment (ofrogon) [inline CONTRIBUTING.md:28]:
```suggestion
Under [packages/](packages/) we have two tools that are deployed on [NPM](https://www.npmjs.com/).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -13,44 +13,36 @@ The following documentation is only for the maintainers of this repository.
...
 
-### PNPM workspace
+## Project overview
 
-This monorepo is using PNPM workspace feature to handle the installation of the npm dependencies and manage the packages interdependencies.
-
-It's important to note that PNPM workspace will **link** the npm dependencies at the root of the workspace. This means that there might not be a *node_modules* directory nested in the packages directories. The npm dependencies are installed in a *node_modules* directory at the root of the workspace and a single *pnpm-lock.yaml* file is generated at the root of the workspace.
-
-## Project Overview
+This project is split into two major sections, [packages/][packages/] and [templates/](templates/).
```
Comment (ofrogon) [inline CONTRIBUTING.md:24]:
```suggestion
This project is split into two major sections, [packages/](packages/) and [templates/](templates/).
```

**Review by ofrogon — DISMISSED** [review]

**Comment by patricklafrance** [comment]
> We should also change the reference to the `master` branch ([line 112](https://github.com/workleap/wl-foundry-cli/pull/58/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R112)) for `main`.

Done!

**Review by ofrogon — APPROVED** [review]

**Pull request merged**